recipes/n88util/build.sh

```diff
@@ -0,0 +1,18 @@
+
+# Create build directory
+mkdir build
+cd build
+BUILD_CONFIG=Release
+
+# CMake
+cmake .. \
+	-G "Ninja" \
+  -DCMAKE_BUILD_TYPE=$BUILD_CONFIG \
+  -DCMAKE_PREFIX_PATH:PATH="${PREFIX}" \
+  -DCMAKE_INSTALL_PREFIX:PATH="${PREFIX}" \
+	-DCMAKE_INSTALL_RPATH:PATH="${PREFIX}/lib" \
+	-DENABLE_TESTING:BOOL=OFF \
+	-DBUILD_SHARED_LIBS:BOOL=ON
+
+# Compile and install!
+ninja install
```

Comment:
Is there something like `make test` which we can run here?

Reply:
Yes! I added `ctest -V` and set `ENABLE_TESTING` on.